test/pkg/environment/common/expectations.go

```diff
@@ -326,35 +327,35 @@ func (env *Environment) ExpectPodsMatchingSelector(selector labels.Selector) []*
 }
 
 func (env *Environment) ExpectUniqueNodeNames(selector labels.Selector, uniqueNames int) {
+	GinkgoHelper()
 	pods := env.Monitor.RunningPods(selector)
 	nodeNames := sets.NewString()
 	for _, pod := range pods {
 		nodeNames.Insert(pod.Spec.NodeName)
 	}
-	ExpectWithOffset(1, len(nodeNames)).To(BeNumerically("==", uniqueNames))
+	Expect(len(nodeNames)).To(BeNumerically("==", uniqueNames))
 }
 
 func (env *Environment) eventuallyExpectScaleDown() {
-	EventuallyWithOffset(1, func(g Gomega) {
+	GinkgoHelper()
+	Eventually(func(g Gomega) {
 		// expect the current node count to be what it was when the test started
 		g.Expect(env.Monitor.NodeCount()).To(Equal(env.StartingNodeCount))
 	}).Should(Succeed(), fmt.Sprintf("expected scale down to %d nodes, had %d", env.StartingNodeCount, env.Monitor.NodeCount()))
 }
 
+
+
 func (env *Environment) EventuallyExpectNotFound(objects ...client.Object) {
-	env.EventuallyExpectNotFoundWithOffset(1, objects...)
+	GinkgoHelper()
+	env.EventuallyExpectNotFoundAssertion(objects...).Should(Succeed())
 }
 
-func (env *Environment) EventuallyExpectNotFoundWithOffset(offset int, objects ...client.Object) {
-	env.EventuallyExpectNotFoundAssertionWithOffset(offset+1, objects...).Should(Succeed())
-}
 
-func (env *Environment) EventuallyExpectNotFoundAssertion(objects ...client.Object) AsyncAssertion {
-	return env.EventuallyExpectNotFoundAssertionWithOffset(1, objects...)
-}
 
-func (env *Environment) EventuallyExpectNotFoundAssertionWithOffset(offset int, objects ...client.Object) AsyncAssertion {
-	return EventuallyWithOffset(offset, func(g Gomega) {
+func (env *Environment) EventuallyExpectNotFoundAssertion(objects ...client.Object) AsyncAssertion {
+	GinkgoHelper()
```

Comment:
Does this need a `GingkgoHelper` since there are no `To()` or `Should()` calls here? This just returns the assertion back to the caller

Reply:
Looks like you didn't pull this out but still resolved this convo. I still think that you need to drop the `GinkgoHelper()` call here